### PR TITLE
[docs] Add note about including heartbeats table in publications

### PR DESCRIPTION
**Pull request by dtseiler**
I think it would be very helpful for the criminally dense like myself to have this explicitly stated in the documentation. What seems obvious to me now comes after nearly a week of frustration.

### CODE REVIEW

**Review by Naros — APPROVED** [review]
@roldanbob, feel free to merge this if the wording is acceptable for you.
Thanks for the PR @dtseiler.

File: documentation/modules/ROOT/pages/connectors/postgresql.adoc [hunk 1]
```diff
@@ -2284,6 +2284,8 @@ Also in the `pg_replication_slots` view, the `restart_lsn` column contains the L
 The database typically reclaims disk space in batch blocks. This is expected behavior and no action by a user is necessary.
 
 * There are many updates in a database that is being tracked but only a tiny number of updates are related to the table(s) and schema(s) for which the connector is capturing changes. This situation can be easily solved with periodic heartbeat events. Set the xref:postgresql-property-heartbeat-interval-ms[`heartbeat.interval.ms`] connector configuration property.
++
+Note that the heartbeat table _must_ be included in your publication in order for the connector to see the heartbeat events and process them. This can be achieved either by adding it explicitly via `ALTER PUBLICATION ... ADD TABLE ...` or by specifying `FOR ALL TABLES` when creating the publication.
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/connectors/postgresql.adoc:2288]:
```suggestion
[NOTE]
====
For the connector to detect and process events from a heartbeat table, you must add the table to the PostgreSQL publication that receives changes streamed from `pgoutput`.  
If the publication already exists before you deploy {prodname}, the connector uses the publications as defined.
If the publication is not configured to automatically replicate changes `FOR ALL TABLES` in the database, you must explicitly add the table to the publication, for example, 

`ALTER PUBLICATION  _<publicationName>_ ADD TABLE _<heartbeatTableName>_;`
====
```

**Review by roldanbob — CHANGES_REQUESTED** [review]
Added the content to an admonition block for enhanced visibility, and added a cross-reference to the `publication.name`configuraiton property.

**Review by roldanbob — APPROVED** [review]

**Comment by dtseiler** [comment]
It looks like this merge only applies to [the "Nightly" set of documentation](https://debezium.io/documentation/reference/nightly/connectors/postgresql.html#postgresql-wal-disk-space). I'm just curious if there's any process or plan for backporting to at least [the current/stable line](https://debezium.io/documentation/reference/stable/connectors/postgresql.html#postgresql-wal-disk-space)?

**Comment by roldanbob** [comment]
Typically, I think that we wouldn't backport documentation changes like this, @dtseiler. But I have no objections if we could spare others the grief that you endured. WDYT @Naros? @jpechane? Any reason not to backport this to 2.3 and/or 2.4?

**Comment by jpechane** [comment]
Backported to 2.4

**Pull request merged**